app/api/file.py

```diff
@@ -0,0 +1,42 @@
+from flask_classy import FlaskView, route
+
+from app.api.service.file import FileService
+from app.api.serializers.file import FileSchema
+
+from flask import request
+
+from flask_apispec.annotations import marshal_with, doc
+
+
+class FileView(FlaskView):
+
+    @route('vault_<vault_id>',)
+    @marshal_with(FileSchema(many=True))
+    @doc(description='Get List of all files, <vault_id> - vault prop')
+    def index(self, vault_id):
+        """List of files"""
+        return FileService.list(vault_id=vault_id)
+
+    @marshal_with(FileSchema)
+    @doc(description='Retrieve one file, <id_> - file prop')
```

Comment:
I'd use `file_id` or even just `id` (unless you want to use the builtin `id()` inside the function)